src/composables/useFlawModel.ts

```diff
@@ -26,11 +27,13 @@ export function useFlawModel(forFlaw: ZodFlawType = blankFlaw(), onSaveSuccess:
   const flaw = ref<ZodFlawType>(forFlaw);
   const cvssScoresModel = useFlawCvssScores(flaw);
   const flawAffectsModel = useFlawAffectsModel(flaw);
+  const flawAttributionsModel = useFlawAttributionsModel(flaw, isSaving, afterSaveSuccess);
   const { wasCvssModified, saveCvssScores } = cvssScoresModel;
   const { affectsToSave, saveAffects, deleteAffects, affectsToDelete } = flawAffectsModel;
 
   const router = useRouter();
   const committedFlaw = ref<ZodFlawType | null>(null);
+  const { saveDraftFlaw } = useDraftFlawStore();
```

Comment:
Is the intention for the store to be read from at some point?

Reply:
Actually `draftFlaw` is being read currently in different places. May be we can remove that and just use the variable assignation.

Reply:
The thing with the store is to get the draft state of the flaw (not saved values) across different components in the application. This is needed cover the problem in bulk actions (like saving multiple references when a flaw is being created) where if one fails in the middle, the rest of draft references from the user are lost.
The options I considered were:

1. Using emitters to get the state in those components
2. Have a store where components can get the draft state

The store looked less complicated to me and reusable in case we need/want to incorporate more flaw fields.

Reply:
Okay, I think this makes sense--do you think we should migrate some of the editing functions over to use this new store? Seems like we should (not in this PR but generally).